src/swagger-specs/query-service.yaml

```diff
@@ -18,2735 +16,4309 @@ info:
-        - $ref: '#/parameters/x-gw-ims-org-id'
-        - $ref: '#/parameters/x-api-key'
-        - $ref: '#/parameters/x-sandbox-name'
-        - $ref: '#/parameters/x-request-id'
-        - $ref: '#/parameters/User-Agent'
-        - $ref: '#/parameters/queries_orderby'
-        - $ref: '#/parameters/limit'
-        - $ref: '#/parameters/start'
-        - $ref: '#/parameters/queries_property'
-        - $ref: '#/parameters/excludeSoftDeleted'
-        - $ref: '#/parameters/excludeHidden'
-        - $ref: '#/parameters/isPrevLink'
-
+      - name: Authorization
+        in: header
+        description: The access token which can be copied from your Experience Platform
+          integration, prefixed with "Bearer " (for example, `Bearer {ACCESS_TOKEN}`).
+          For more information on how to obtain this value, visit the documentation
+          covering making API calls in the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+        required: true
+        schema:
+          type: string
+      - name: x-gw-ims-org-id
+        in: header
+        description: The Organization ID which can be copied from your Experience
+          Platform integration. For more information on how to obtain this value,
+          visit the documentation covering making API calls in the [authentication
+          tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+        required: true
+        schema:
+          type: string
+      - name: x-api-key
+        in: header
+        description: The API key which can be copied from your Experience Platform
+          integration. For more information on how to obtain this value, visit the
+          documentation covering making API calls in the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+        required: true
+        schema:
+          type: string
+      - name: x-sandbox-name
+        in: header
+        description: The name of the sandbox in which the operation will take place.
+          See the [sandboxes overview documentation](https://adobe.com/go/sandbox-overview-en)
+          for more information.
+        required: true
+        schema:
+          type: string
+      - name: x-request-id
+        in: header
+        description: An optional header that serves as a tracking ID for the request.
+          If one is not provided on the request, the server generates one automatically.
+        schema:
+          type: string
+      - name: User-Agent
+        in: header
+        description: The type of the client making the HTTP request.
+        schema:
+          type: string
+      - name: orderby
+        in: query
+        description: 'Field to order results by. Supported fields: created, updated.
+          Prepend property name with + for ASC,- for DESC order. Default is -created.
+          Note that the plus sign (+) has to be escaped with %2B. Example: %2Bcreated
+          for ASC created.'
```

Comment:
Is there a way to better emphasize the examples here? would backticks work around -created and %2Bcreated?